From 6bd728f9133dd8fa9d5c9d90c31beb370a5bcf82 Mon Sep 17 00:00:00 2001
From: Prasanth Jayachandran <prasanthj@apache.org>
Date: Tue, 18 Oct 2016 18:41:49 -0700
Subject: [PATCH 1037/1164] CDH-48309 HIVE-14974: TestBeeLineHistory throws
 NPE in ShutdownHook (Prasanth Jayachandran
 reviewed by Siddharth Seth)

(cherry picked from commit 044f1fa3ce868d4880adbe8c92a24e15e8b88dba)

Change-Id: I65d6e25b2b7c0ebd9f00606a1b5029973006acf0
---
 .../apache/hive/beeline/TestBeeLineHistory.java    |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/beeline/src/test/org/apache/hive/beeline/TestBeeLineHistory.java b/beeline/src/test/org/apache/hive/beeline/TestBeeLineHistory.java
index e861976..7c892fd 100644
--- a/beeline/src/test/org/apache/hive/beeline/TestBeeLineHistory.java
+++ b/beeline/src/test/org/apache/hive/beeline/TestBeeLineHistory.java
@@ -33,7 +33,7 @@
  */
 public class TestBeeLineHistory {
 
-  private static final String fileName = "history";
+  private static final String fileName = System.getProperty("test.tmp.dir") + "/history";
 
   @BeforeClass
   public static void beforeTests() throws Exception {
-- 
1.7.9.5

